### PR TITLE
Feature: DAO 555 - list item text

### PR DESCRIPTION
**Pull request by WissenIstNacht**
This PR implements
- ListItem->Text component
- Corresponding Stories
- Corresponding Tests

### CODE REVIEW

**Review by RakeshUP — APPROVED** [review]
Looks good!

File: packages/ui-components/src/components/listItem/text.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import styled from 'styled-components';
+import {IconType} from '../icons';
+
+type CustomButtonProps = Omit<
+  ButtonHTMLAttributes<HTMLButtonElement>,
+  'disabled'
+>;
+export type ListItemTextProps = CustomButtonProps & {
+  /**
+   * State that can be explisitely set by the client. These are mutually
```
Comment (sepehr2github) [inline packages/ui-components/src/components/listItem/text.tsx]:
```suggestion
   * State that can be explicitly set by the client. These are mutually
```

Comment (sepehr2github) [reply packages/ui-components/src/components/listItem/text.tsx:11]: Just a simple misspelling.

**Review by sepehr2github — APPROVED** [review]
Awesome! Thank you. Just a simple misspelling. ;)

**Pull request merged**